proxy/http/remap/unit-tests/nexthop_test_stubs.cc

```diff
@@ -125,6 +125,14 @@ build_request(int64_t sm_id, HttpSM *sm, sockaddr_in *ip, const char *os_hostnam
   sm->t_state.txn_conf               = reinterpret_cast<OverridableHttpConfigParams *>(_my_txn_conf);
 }
 
+void
+GetConfigInteger(int *v, const char *n)
+{
+  if (strncmp(n, "proxy.config.http.parent_proxy.max_trans_retries", strlen(n)) == 0) {
```

Comment:
Why not just use strcmp()?

Reply:
@ywkaras I think strncmp() is safer.  Why do you prefer strcmp()?

Reply:
I think the safety issue is that a C string might have a missing null terminator.  But strncmp() only helps with  that if it's got a constant as the length.  strcmp() is a safe as strncmp() if one of the strings is a string literal.

Reply:
@ywkaras I made the change to strcmp() and added some nullptr checks.